Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -438,6 +527,13 @@ namespace xr
             swapchainCreateInfo.createFlags = createFlags;
             swapchainCreateInfo.usageFlags = usageFlags;
 
+            XrSecondaryViewConfigurationSwapchainCreateInfoMSFT secondaryViewConfigCreateInfo{
+                XR_TYPE_SECONDARY_VIEW_CONFIGURATION_SWAPCHAIN_CREATE_INFO_MSFT};
+            if (HmdImpl.Extensions->SecondaryViewConfigurationSupported) {
```

Comment:
nit: Style, this document has condition block braces on newlines. We might should consider adding a clang format to this dependency just to make it easy to run auto-format.

Reply:
I have updated declarations for secondary view types